tests/Feature/Controller/Student/StudentAdditionalTrainingListControllerTest.php

```diff
@@ -8,60 +8,72 @@
 use App\Models\Student;
 use App\Models\Resume;
 use App\Models\AdditionalTraining;
-use Tests\Fixtures\Students;
+use Illuminate\Foundation\Testing\DatabaseTransactions;
 use App\Http\Controllers\api\Student\StudentAdditionalTrainingListController;
-use App\Service\Student\StudentAdditionalTrainingListService;
 
 class StudentAdditionalTrainingListControllerTest extends TestCase
 {
-    public function testStudentAdditionalTrainingListControllerReturnsAdditionalTrainingDetailsForValidStudentUuid()
-    {
-        $student = Student::factory()->create();
-
-        $resume = Resume::factory()->create(['student_id' => $student->id]); 
+    
+    use DatabaseTransactions;
+    protected $student;
+    protected $additionalTrainings;
 
-        $additionalTraining1 = AdditionalTraining::factory()->create();
+    protected function setUp(): void
+    {
+        parent::setUp();
 
-        $additionalTraining2 = AdditionalTraining::factory()->create();
+        $this->student = Student::factory()->create();
 
-        $resume->additionalTrainings()->attach([$additionalTraining1->id, $additionalTraining2->id]);
+        $this->additionalTrainings = AdditionalTraining::factory()->count(2)->create();
 
-        $response = $this->getJson(route('student.additionaltraining', ['studentId' => $student->id]));
+        $this->student->resume()->create([
+            'additional_training_ids' => json_encode($this->additionalTrainings->pluck('id')->toArray())
+        ]);
+    }
+    
+    public function testCanFindAdditionalTrainingDetails()
+    {        
+        $response = $this->get(route('student.additionaltraining', ['student' => $this->student]));      
 
         $response->assertStatus(200);
 
-        $response->assertJsonStructure(['additional_trainings']);
+        $response->assertJsonStructure([            
+                '*' => [
+                    'id',
+                    'course_name',
+                    'study_center',
+                    'course_beginning_year',
+                    'course_ending_year',
+                    'duration_hrs'
+                ]
+            ]
+        );
     }
 
-    public function testStudentAdditionalTrainingListControllerReturns_404ForInvalidStudentUuid()
+    public function testCanReturn_404WhenStudentIsNotFound()
```

Comment:
remove underscores if you are using camelcase

Reply:
done